core/framework/doc/index.rst

```diff
@@ -82,4 +82,4 @@ With the use of adapters and the selection service, each application simply need
 
 Register zero or more
 `AdapterFactories <https://github.com/ControlSystemStudio/phoebus/blob/master/app/logbook/ui/src/main/java/org/phoebus/logbook/ui/adapters/AppSelectionAdapterFactory.java>`_
-which provide the mechanism to adapt an AppSelection to a simple LogEntry or an Olog LogEnrty
\ No newline at end of file
+which provide the mechanism to adapt an AppSelection to a simple LogEntry or an Olog LogEnrty
```

Comment:
While at it, can you fix `LogEnrty` -> `LogEntry`?

Reply:
It looks like you guys have `docs/core/` and a bunch of other documentation files under the git ignore:
https://github.com/ControlSystemStudio/phoebus/blob/master/docs/source/.gitignore

Are the contents of these files auto-generated by something?